bemuse/src/online/level.ts

```diff
@@ -0,0 +1,21 @@
+import { MappingMode, isMappingMode } from 'bemuse/app/entities/Options'
```

Comment:
Unfortunately this introduces a circular dependency between `bemuse/app` and `bemuse/online`. It is intended for the direct subdirectories in `bemuse/src` to be like an npm package (so-called [packlets](https://www.npmjs.com/package/@rushstack/eslint-plugin-packlets)), so they shouldn’t create a circular dependency between each other.

(I haven’t added a circular dependency detecter to the project yet, so there is currently no automated checks for this, sorry.)

I would recommend one of these:

- Duplicate `MappingMode` into `PlayMode` here.
- Move `MappingMode` and `isMappingMode` into `bemuse/rules`.

Reply:
I moved it into `bemuse/rules` on cd1225d. Thank you for reviewing.